packages/cli/src/main/kotlin/elide/tool/cli/cmd/repl/ToolShellCommand.kt

```diff
@@ -1513,16 +1453,11 @@ import org.graalvm.polyglot.Engine as VMEngine
       )
     }
 
-    withVM(context, userBundleUris, bundleUris, hostIO = hostIo, accessControl::apply) {
-      // warn about experimental status, as applicable
-      require(!initialized.get()) {
```

Comment:
No need to worry about concurrency in initialization, the engine is now created lazily so it can never be used before initialization, nor can it be re-configured (the API provides no way to do this).